tests/integration/galley/conversion_test.go

```diff
@@ -57,9 +57,9 @@ func TestConversion(t *testing.T) {
 						// Do init for the first set. Use Meshconfig file in this set.
 						if i == 0 {
 							if fset.HasMeshConfigFile() {
-								mc, err := fset.LoadMeshConfigFile()
-								if err != nil {
-									t.Fatalf("Error loading Mesh config file: %v", err)
+								mc, er := fset.LoadMeshConfigFile()
+								if er != nil {
```

Comment:
Is this standard compiler/linter complaining about these?

Reply:
This is Goland rules mostly. It makes harder to go through files and fix things, so I decided to clean things up a bit as I go.